index.html

```diff
@@ -89,5 +93,147 @@ <h3 class="subtitle">I am a Software Developer</h3>
+          <li class="tdisplay">Ruby</li>
+        </ul>
+        <button class="btstyle pjbutton" type="button">See Project</button>
+      </div>
+      <div class="project2 proj">
+        <h2 class="ptitle">Professional Art Printing Data</h2>
+        <p class="pbox">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="ptech">
+          <li class="tdisplay">html</li>
+          <li class="tdisplay">bootstrap</li>
+          <li class="tdisplay">Ruby</li>
+        </ul>
+        <button class="btstyle pjbutton" type="button">See Project</button>
+      </div>
+      <div class="project3 proj">
+        <h2 class="ptitle">Professional Art Printing Data</h2>
+        <p class="pbox">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="ptech">
+          <li class="tdisplay">html</li>
+          <li class="tdisplay">bootstrap</li>
+          <li class="tdisplay">Ruby</li>
+        </ul>
+        <button class="btstyle pjbutton" type="button">See Project</button>
+      </div>
+      <div class="project4 proj">
+        <h2 class="ptitle">Professional Art Printing Data</h2>
+        <p class="pbox">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="ptech">
+          <li class="tdisplay">html</li>
+          <li class="tdisplay">bootstrap</li>
+          <li class="tdisplay">Ruby</li>
+        </ul>
+        <button class="btstyle pjbutton" type="button">See Project</button>
+      </div>
+      <div class="project5 proj">
+        <h2 class="ptitle">Professional Art Printing Data</h2>
+        <p class="pbox">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="ptech">
+          <li class="tdisplay">html</li>
+          <li class="tdisplay">bootstrap</li>
+          <li class="tdisplay">Ruby</li>
+        </ul>
+        <button class="btstyle pjbutton" type="button">See Project</button>
+      </div>
+      <div class="project6 proj">
+        <h2 class="ptitle">Professional Art Printing Data</h2>
+        <p class="pbox">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="ptech">
+          <li class="tdisplay">html</li>
+          <li class="tdisplay">bootstrap</li>
+          <li class="tdisplay">Ruby</li>
+        </ul>
+        <button class="btstyle pjbutton" type="button">See Project</button>
+      </div>
+    </section>
+    <!--about me-->
+    <section class="aboutme">
+      <h1 class="title abtitle">About me</h1>
```

Comment:
- [ ] Kindly use an `h2` tag here as your page should have only one `h1` tag which should be the main heading of your page 👍

Reply:
Removed every h1 but the first one!